### PR TITLE
feat(contracts): add withdraw account to validator

### PR DESCRIPTION
**Pull request by 0xHansLee**
Added withrdraw account field to validator.

### CODE REVIEW

File: packages/contracts/contracts/test/CommonTest.t.sol [hunk 1]
```diff
@@ -254,6 +254,7 @@ contract L2OutputOracle_Initializer is UpgradeGovernor_Initializer {
     // Test data
     address internal asserter = 0x000000000000000000000000000000000000aAaB;
     address internal challenger = 0x000000000000000000000000000000000000AAaC;
+    address internal withdrawAcc = 0x000000000000000000000000000000000000AaaD;
```
Comment (seolaoh) [inline packages/contracts/contracts/test/CommonTest.t.sol:257]: Why not change this address to be distinct from guardian address?

File: packages/contracts/contracts/test/ValidatorManager.t.sol [hunk 2]
```diff
@@ -210,7 +210,7 @@ contract ValidatorManagerTest is ValidatorSystemUpgrade_Initializer {
         emit ValidatorActivated(trusted, block.timestamp);
         vm.expectEmit(true, true, false, true, address(valMgr));
         emit ValidatorRegistered(trusted, true, commissionRate, commissionMaxChangeRate, assets);
-        valMgr.registerValidator(assets, commissionRate, commissionMaxChangeRate);
+        valMgr.registerValidator(assets, commissionRate, commissionMaxChangeRate, withdrawAcc);
         vm.stopPrank();
 
         assertEq(assetToken.balanceOf(trusted), trustedBalance - assets);
```
Comment (seolaoh) [inline packages/contracts/contracts/test/ValidatorManager.t.sol:216]: Please add the check that withdraw account is set properly.

**Comment by seolaoh** [comment]
The subcommand should be changed also.

File: packages/contracts/contracts/test/CommonTest.t.sol (hunk 1 above)
Comment (0xHansLee) [reply packages/contracts/contracts/test/CommonTest.t.sol:257]: Oh, I didn't know it's the same with guardian address. I'll change it.

**Comment by seolaoh** [comment]
Can you squash https://github.com/kroma-network/kroma/pull/343/commits/93fe0a8b93068470c2fb4945e5fb2ba6cd0a0802, https://github.com/kroma-network/kroma/pull/343/commits/e3b96f46c8def2d680adce50b180cfd3411f3fa0 to the last commit?

**Comment by 0xHansLee** [comment]
> Can you squash https://github.com/kroma-network/kroma/commit/93fe0a8b93068470c2fb4945e5fb2ba6cd0a0802, https://github.com/kroma-network/kroma/commit/e3b96f46c8def2d680adce50b180cfd3411f3fa0 to the last commit? Or are these commits intended?

It is automatically added to this branch by clicking `commit suggestion` in github web. I will squash and merge this PR to base branch. Is it ok?

File: kroma-validator/cmd/validator/cmd.go [hunk 3]
```diff
@@ -173,13 +173,14 @@ func Register(ctx *cli.Context) error {
 
 	commissionRate := uint8(ctx.Uint64("commission-rate"))
 	commissionMaxChangeRate := uint8(ctx.Uint64("commission-max-change-rate"))
+	withdrawAccount := common.HexToAddress(ctx.String("withdraw-account"))
```
Comment (seolaoh) [inline kroma-validator/cmd/validator/cmd.go:176]:
```suggestion
	withdrawAccount := opservice.ParseAddress(ctx.String("withdraw-account"))
```

File: kroma-validator/cmd/validator/cmd.go [hunk 4]
```diff
@@ -173,13 +173,14 @@ func Register(ctx *cli.Context) error {
...
 
 	valMgrAbi, err := bindings.ValidatorManagerMetaData.GetAbi()
 	if err != nil {
 		return fmt.Errorf("failed to get ValidatorPool ABI: %w", err)
```
Comment (seolaoh) [inline kroma-validator/cmd/validator/cmd.go:180]:
```suggestion
		return fmt.Errorf("failed to get ValidatorManager ABI: %w", err)
```

**Review by seolaoh — APPROVED** [review]
LGTM

**Pull request merged**